src/opera/test/pge/dswx_ni/test_dswx_ni_pge.py

```diff
@@ -389,6 +390,90 @@ def test_dswx_ni_pge_output_validation(self):
             if os.path.exists(test_runconfig_path):
                 os.unlink(test_runconfig_path)
 
+    def test_dswx_ni_pge_validate_output_product_filenames(self):
+        """Test the _validate_output_product_filenames() method in DSWxNIPostProcessorMixin."""
+        # for the nominal test use the output file names from the test runconfig file.
+        runconfig_path = join(self.data_dir, 'test_dswx_ni_config.yaml')
+
+        with open(runconfig_path, 'r', encoding='utf-8') as stream:
+            runconfig_dict = yaml.safe_load(stream)
+
+        with open(runconfig_path, 'w', encoding='utf-8') as outfile:
+            yaml.safe_dump(runconfig_dict, outfile, sort_keys=False)
+
+        pge = DSWxNIExecutor(pge_name="DSWxNIPgeTest", runconfig_path=runconfig_path)
+
+        pge.run_preprocessor()
```

Comment:
Let's add a comment here that we run only the preprocessor to parse and ingest the RunConfig

Reply:
Done